l10n_br_fiscal/models/operation_line.py

```diff
@@ -216,6 +217,12 @@ def map_fiscal_taxes(
         ):
             self._build_mapping_result(mapping_result, tax_definition)
 
+        if company.tax_framework in TAX_FRAMEWORK_SIMPLES_ALL:
+            for tax_definition in self.tax_definition_ids.map_tax_definition(
+                company, partner, product, ncm=ncm, nbm=nbm, nbs=nbs, cest=cest
+            ):
+                self._build_mapping_result(mapping_result, tax_definition)
+
```

Comment:
Ola @marcelsavegnago eu acho essa contribuição boa. A unica coisa é que eu re-organisaria o codigo de forma um pouco diferente:

esse bloco depois:
```
        # 2 From NCM
        if not ncm and product:
            ncm = product.ncm_id
```

Eu jogaria ele dentro do bloco `if company.tax_framework == TAX_FRAMEWORK_NORMAL:` ja que esse ncm esta usado apenas la, imediatamente depois do if.

Segundo, agora que esse `if company.tax_framework == TAX_FRAMEWORK_NORMAL:` seguiria o bloco do seu `if company.tax_framework in TAX_FRAMEWORK_SIMPLES_ALL:` eu transformaria então o segundo if em elif para deixar o código mais explicito:

`elif company.tax_framework == TAX_FRAMEWORK_NORMAL:`

O que vc acha?

Reply:
cc @renatonlima

Reply:
@rvalyi não seria melhor deixar o este trecho do NCM antes deste if ? ` if company.tax_framework in TAX_FRAMEWORK_SIMPLES_ALL:` e com isso permitir que o NCM influencie tanto em empresas normal como simples.